test/server/config/stats/config_test.cc

```diff
@@ -0,0 +1,89 @@
+#include <string>
+
+#include "envoy/registry/registry.h"
+
+#include "common/protobuf/utility.h"
+#include "common/stats/statsd.h"
+
+#include "server/config/stats/statsd.h"
+
+#include "test/mocks/server/mocks.h"
+
+#include "api/bootstrap.pb.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+using testing::NiceMock;
+using testing::Return;
+using testing::ReturnRef;
+using testing::_;
```

Comment:
Nit: place outside the namespace block as per Envoy convention.

Reply:
There are a few other tests (including the one I modeled after) that put it inside the Envoy namespace block.  Should I go ahead and update those as well?

Reply:
Yep, thanks.